### ISSUE TITLE
BBS02 Config Write Error

### ISSUE THREAD

**Issue by MrHumanRebel** [danielnilsson9/bbs-fw#108]
Any ideas? 
Tried connecting, rebooting.
The firmware flashing process was flawless.

![image](https://user-images.githubusercontent.com/80947401/231727786-e016d010-742e-4bd9-af31-ee104e721edc.png)

**Comment by kb79000** [comment]
When you are logged in with the screen do you have an error code?

**Comment by danielnilsson9** [comment]
No idea what is causing this, sounds like some interference, it works correctly if you are not charging the battery?

**Comment by MrHumanRebel** [comment]
Yes indeed! If the battery is not charging, it works fine, I even took a short trip with the new FW.

**Comment by danielnilsson9** [comment]
Interesting, only thing I can think of it the charger producing a ripple which is interfering with the serial lines which runs in the same cable and could easily induce a voltage (noise) on the rx/tx lines given the length of the exposure and no shielding.

(B+ and B- runs inte main harness and display cable all the way up to the display)

Not likely to be a firmware bug in my opinion

**Comment by MrHumanRebel** [comment]
That would be a reasonable explanation to this, I have no experience in the cable routing inside these motors. 
It may be good idea to include this finding to the documentation, to warn potential new users doing stuff to the controller while charging to avoid firmware corruption.
